model/generate.go

```diff
@@ -226,7 +226,8 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, cachedProj
 	for _, bv := range g.BuildVariants {
 		newTVPairs = appendTasks(newTVPairs, bv, p)
 	}
-	newTVPairs.ExecTasks = append(newTVPairs.ExecTasks, IncludePatchDependencies(p, newTVPairs.ExecTasks)...)
+	newTVPairs.ExecTasks = IncludePatchDependencies(p, newTVPairs.ExecTasks)
+	//newTVPairs.ExecTasks = append(newTVPairs.ExecTasks, IncludePatchDependencies(p, newTVPairs.ExecTasks)...)
```

Comment:
There's no need to keep old code around since we have the commit history

Reply:
oops.
removed.